docs/install/vendor-options.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Commercial Knative Offerings"
```

Comment:
maybe instead of using "Commercial" we can use something more generic to cover any project (ie open source, free tool) or product (payed, freeish).

Maybe "Downstream Kantive distributions"

Reply:
Yeah, we definitely need a place to document things like this because there are a good number, and it will just get harder to track as the list grows.

I wanted to orient this more towards "install", and have the vendor docs link to "How do I hold the product?"  Perhaps we should have something like: https://github.com/tektoncd/friends?